src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1504,6 +1506,35 @@ public function normalizeId($id)
         return isset($this->definitions[$id]) || isset($this->aliasDefinitions[$id]) || isset($this->removedIds[$id]) ? $id : parent::normalizeId($id);
     }
 
+    /**
+     * Gets removed binding ids.
+     *
+     * @return array
+     *
+     * @internal
+     */
+    public function getRemovedBindingIds()
+    {
+        return $this->removedBindingIds;
+    }
+
+    /**
+     * Adds a removed binding id.
+     *
+     * @param int $id
+     *
+     * @internal
+     */
+    public function addRemovedBindingIds($id)
```

Comment:
`int` instead of annotation+docblock title

Reply:
This should actually be `string`.

Reply:
good catch, could you please send a PR to fix it?

Reply:
https://github.com/symfony/symfony/pull/31173